charts/hcloud-fip-controller/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: hcloud-fip-controller
+description: hcloud-fip-controller is a small controller, to handle floating IP management in a kubernetes cluster on hetzner cloud virtual machines.
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
```

Comment:
I'd like to use either version==appVersion or at least an equal minor version

Reply:
That's not going to work. What will be the chart version when the chart needs an update but the app version stays the same? What will be the chart version when the chart undergoes a major change but the app version stays the same?